t/.perlcritic-history

```diff
@@ -1,141 +1,144 @@
-            'Perl::Critic::Policy::Subroutines::RequireArgUnpacking' => 0,
-            'Perl::Critic::Policy::RegularExpressions::RequireBracesForMultiline' => 0,
-            'Perl::Critic::Policy::Subroutines::ProhibitUnusedPrivateSubroutines' => 0,
-            'Perl::Critic::Policy::Documentation::RequirePodAtEnd' => 4,
-            'Perl::Critic::Policy::Freenode::StrictWarnings' => 0,
-            'Perl::Critic::Policy::Variables::ProtectPrivateVars' => 0,
-            'Perl::Critic::Policy::Freenode::DeprecatedFeatures' => 0,
-            'Perl::Critic::Policy::BuiltinFunctions::ProhibitBooleanGrep' => 6,
-            'Perl::Critic::Policy::Freenode::Prototypes' => 5,
+            'Perl::Critic::Policy::Subroutines::ProhibitNestedSubs' => 1,
+            'Perl::Critic::Policy::BuiltinFunctions::ProhibitReverseSortBlock' => 20,
+            'Perl::Critic::Policy::RegularExpressions::ProhibitUselessTopic' => 18,
+            'Perl::Critic::Policy::Freenode::ConditionalDeclarations' => 0,
+            'Perl::Critic::Policy::Freenode::Each' => 0,
+            'Perl::Critic::Policy::TestingAndDebugging::RequireUseWarnings' => 4,
+            'Perl::Critic::Policy::Modules::ProhibitExcessMainComplexity' => 36,
+            'Perl::Critic::Policy::Subroutines::ProhibitSubroutinePrototypes' => 10,
+            'Perl::Critic::Policy::ValuesAndExpressions::ProhibitLeadingZeros' => 24,
+            'Perl::Critic::Policy::InputOutput::ProhibitJoinedReadline' => 0,
+            'Perl::Critic::Policy::ValuesAndExpressions::RequireQuotedHeredocTerminator' => 63,
             'Perl::Critic::Policy::Freenode::ConditionalImplicitReturn' => 0,
-            'Perl::Critic::Policy::BuiltinFunctions::ProhibitReverseSortBlock' => 0,
-            'Perl::Critic::Policy::Freenode::EmptyReturn' => 0,
-            'Perl::Critic::Policy::ClassHierarchies::ProhibitAutoloading' => 0,
-            'Perl::Critic::Policy::Freenode::Threads' => 0,
-            'Perl::Critic::Policy::Freenode::BarewordFilehandles' => 2,
-            'Perl::Critic::Policy::Subroutines::ProhibitReturnSort' => 0,
-            'Perl::Critic::Policy::ValuesAndExpressions::ProhibitQuotesAsQuotelikeOperatorDelimiters' => 0,
-            'Perl::Critic::Policy::Variables::RequireNegativeIndices' => 0,
-            'Perl::Critic::Policy::InputOutput::RequireBriefOpen' => 5,
+            'Perl::Critic::Policy::ValuesAndExpressions::RequireUpperCaseHeredocTerminator' => 0,
+            'Perl::Critic::Policy::Variables::ProhibitReusedNames' => 144,
+            'Perl::Critic::Policy::Freenode::Wantarray' => 0,
+            'Perl::Critic::Policy::ValuesAndExpressions::ProhibitLongChainsOfMethodCalls' => 16,
+            'Perl::Critic::Policy::Modules::RequireEndWithOne' => 214,
+            'Perl::Critic::Policy::BuiltinFunctions::ProhibitBooleanGrep' => 229,
+            'Perl::Critic::Policy::Miscellanea::ProhibitUnrestrictedNoCritic' => 0
```

Comment:
📈

Reply:
> 📈
[](
# Clone my vim settings to a .vim directory
git clone git@github.com:lucasr/vim-settings.git .vim

# Link standard vim settings file to the one .vim directory
ln -s .vim/vimrc ~/.vimrc

# Fetch all bundle submodules
cd .vim
- git submodules update --init)